### PR TITLE
Updated FeatureExtraction.py

### PR DESCRIPTION
**Pull request by slee172**
FeatureExtraction.py is modified.

1 Arbitary sized arrays can be used as parameters.

2 Stand matrix is removed.

3 ValueError is fixed.

4 ZeorDivisionError is fixed.

5 Boundary value error is fixed.

### CODE REVIEW

File: histomicstk/features/FeatureExtraction.py [hunk 1]
```diff
@@ -21,11 +21,6 @@ def FeatureExtraction(Label, In, Ic, W, K=128, Fs=6, Delta=8):
         A T x T intensity image for Nuclei.
```
Comment (cdeepakroy) [inline histomicstk/features/FeatureExtraction.py]: Works only on square images? If not this should be changed.

File: histomicstk/features/FeatureExtraction.py [hunk 2]
```diff
@@ -21,11 +21,6 @@ def FeatureExtraction(Label, In, Ic, W, K=128, Fs=6, Delta=8):
...
     Ic : array_like
         A T x T intensity image for Cytoplasms.
```
Comment (cdeepakroy) [inline histomicstk/features/FeatureExtraction.py]: Works only on square images? If not this should be changed.

**Comment by cdeepakroy** [comment]
@slee172 
* Few minor comments in the docstring. 
* Integrate changes from master into your branch.

Rest LGTM!

File: histomicstk/features/FeatureExtraction.py (hunk 1 above)
Comment (slee172) [inline histomicstk/features/FeatureExtraction.py]: It supports M by N images. My mistake. Modified to M by N.

**Pull request merged**